innovation_sweet_spots/getters/crunchbase.py

```diff
@@ -89,3 +89,9 @@ def get_crunchbase_acquisitions() -> pd.DataFrame:
     return pd.read_csv(CB_PATH / "crunchbase_acquisitions.csv").pipe(
         restore_column_names
     )
+
+
+def get_crunchbase_gtr_lookup() -> pd.DataFrame:
+    """Table with UK Crunchbase organisation ids with fuzzy
+    matched Gateway to Research organisation ids"""
+    return pd.read_csv(CB_GTR_LINK_PATH / "cb_gtr_id_lookup.csv", index_col=0)
```

Comment:
I only found `cb_gtr_id_lookup_test.csv` on the s3. I guess the issue is that I haven't run the the production mdoe of the linking code?

Reply:
I have uploaded the `cb_gtr_id_lookup_test.csv` file onto S3. I will rerun creating `cb_gtr_id_lookup_test.csv` when/if the GtR files get updated.